mantaray/views.py

```diff
@@ -542,6 +547,18 @@ def on_part(self, nick: str, reason: str | None) -> None:
             show_in_gui=self.server_view.should_show_join_leave_message(nick),
         )
 
+    def on_kick(self, channel:str, nick:str, reason: str | None) -> None:
+        self.userlist.remove_user(nick)
+        if reason is None:
+            extra = ''
+        else:
+            extra = " " + reason + ""
+        self.add_message(
+            "*",
+            (nick, ["other-nick"]),
+            (" has been kicked from the channel." + f'     (Reason: {extra})', []),
+            show_in_gui=self.server_view.should_show_join_leave_message(nick))
```

Comment:
I think we always want to display kick messages. They are rare and likely won't get very annoying. You can remove this line, because `show_in_gui` defaults to `True`.